tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -417,10 +417,13 @@ def start_cmd(self, node):
         else:
             cmd += " --bootstrap-server %s" % self.kafka.bootstrap_servers(self.security_config.security_protocol)
 
-        cmd += " --reset-policy %s --group-id %s --topic %s --session-timeout %s" % \
-               (self.reset_policy, self.group_id, self.topic,
-                self.session_timeout_sec*1000)
-               
+        cmd += " --reset-policy %s --group-id %s --topic %s" % \
+                (self.reset_policy, self.group_id, self.topic)
+
+        # session timeout is not supported when using CONSUMER group protocol
+        if self.session_timeout_sec > 0 and self.is_consumer_group_protocol_enabled():
```

Comment:
this will swallow the incorrect configs in e2e. We can just set the session timeout if e2e test case does set it.

Reply:
Only `VerifiableConsumerTest` specifically set session timeout, but I think it's a bit redundant, so I refactor them out.